### DISCUSSION TITLE
delete scrobbles...

### DISCUSSION THREAD

**Discussion by Zxyahi** [web-scrobbler/web-scrobbler#3583] [Q&A]
how delete scrobbles made by web scrobbler?

can someone help me?

**Comment by pixiekat** [comment]
You can't do it from the extension but you can do it from your last.fm library page by clicking on the three vertical dots when hovering over a scrobble and selecting delete. 

![Screenshot_1](https://user-images.githubusercontent.com/32232155/221414810-ad30df9d-44d4-4c1c-8a9a-fb2881de7c3e.png)
